python/cudf/cudf/utils/utils.py

```diff
@@ -31,6 +34,7 @@
     "__ge__",
 }
 
+_NVTX_COLORS = ["green", "blue", "purple", "rapids"]
```

Comment:
What color is "rapids"? Is it not very similar to purple?

Reply:
Yeah, those both aren't similar:
https://github.com/NVIDIA/NVTX/blob/eb0d6301e833fa3631d774272f09741582d29e94/python/nvtx/colors.py#L14-L15